README.md

```diff
@@ -16,9 +16,37 @@ The models can all be used in the same way, using `fit()` and `predict()` functi
 similar to scikit-learn. The library also makes it easy to backtest models,
 and combine the predictions of several models and external regressors.
 
+## Table of Content
+
+- [Darts: Easy manipulation and forecasting of time series.](#darts-easy-manipulation-and-forecasting-of-time-series)
```

Comment:
I really like this table of contents, but I feel like the first entry (Darts: Easy manipulation...) is not really necessary.

Reply:
I agree but it is automatically generated by VScode and follows the changes we make I will try to remove it but we might lose that automation feature in the process

Reply:
Ahh I see. If it means losing the automatic generation, it might not be worth it

Reply:
I would actually vote for not having the best-possible looking readme, and automation should come second.

Reply:
ToC has been removed (until better solution that works both in github md, and sphinx rst)